index.html

```diff
@@ -90,6 +92,227 @@ <h2>Let's connect</h2>
+                                </svg>
+
+                            </a>
+                        </li>
+                    </ul>
+
+                    <div class="mt-1">
+                        <a class="btn" href="#">Get my resume</a>
+                    </div>
+
+                </section>
+
+                <ul class="drop-buttons">
+                    <li class="drop-button">
+                        <span class="title">Languages</span>
+                        <ul class="languages extra-menu">
+                            <li>
+                                <img src="../images/javascript.png" alt="javascript icon" />
+                                <span>JavaScript</span>
+                            </li>
+                            <li>
+                                <img src="../images/html.png" alt="javascript icon" />
+                                <span>HTML</span>
+                            </li>
+                            <li>
+                                <img src="../images/css.png" alt="javascript icon" />
+                                <span>CSS</span>
+                            </li>
+                        </ul>
+                    </li>
+
+                    <li class="drop-button">
+                        <span class="title">Frameworks</span>
+                    </li>
+
+                    <li class="drop-button">
+                        <span class="title">Skills</span>
+
+                    </li>
+                </ul>
```

Comment:
- [ ] Well done on doing the dropdown button, I would have approved it but the icons don't open as expected. please 🙏🏽  review the path of the image to correct the problem and look similar to the Figma template. Also, kindly change the `alt` attribute to have a different description according to the image.

![image](https://user-images.githubusercontent.com/58642949/219907617-0015ad8f-e32d-4c59-bb2b-e12020e50aad.png)

Reply:
![workingFine](https://user-images.githubusercontent.com/88587370/219907983-08c5615a-1633-444d-9e2f-a8df758820e3.png)

I don´t get it. Mine is working fine without changes

Reply:
![workingfine2](https://user-images.githubusercontent.com/88587370/219908045-4da3f8e1-b413-42b6-8d33-6a4d4faa9ff7.png)
@franclobo

Reply:
okay, I see my mistake now. I was accessing it from the css folder because other icon somewhere were accessed from the CSS instead of the index